ui/src/components/App.spec.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import { createRenderer } from 'react-test-renderer/shallow';
-import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
```

Comment:
```suggestion
import { App } from './App';
```
This will import just the App component without Redux (after the above change to export `const App` is made in `App.js`